source/includes/administration/_notifications.md

```diff
@@ -0,0 +1,374 @@
+
+```json
+{
+    "data": [
+        {
+            "publishedAt": "2020-10-29T19:27:34.299Z",
+            "authorId": "15419d47-0ec4-47d5-8622-4fbb1948844f",
+            "version": 1,
+            "createdAt": "2020-10-29T15:04:36.134Z",
+            "authorLastName": "Root",
+            "authorFirstName": "Root",
+            "translations": [
+                {
+                    "language": "en",
+                    "id": "99f815c6-2aba-4462-9419-63daa67712e2",
+                    "text": "system-notification",
+                    "type": "title",
+                    "version": 1
+                },
+                {
+                    "language": "fr",
+                    "id": "812f3b76-bc6b-4816-ace8-3e69c6e1efcd",
+                    "text": "en français",
+                    "type": "title",
+                    "version": 1
+                },
+                {
+                    "language": "en",
+                    "id": "94c4bedb-df83-49f0-beb3-8e672e28c316",
+                    "text": "system",
+                    "type": "body",
+                    "version": 1
+                },
+                {
+                    "language": "fr",
+                    "id": "98c1f48c-1f2b-4dc6-a43c-3f239a2473a7",
+                    "text": "system",
+                    "type": "body",
+                    "version": 1
+                }
+            ],
+            "deprecatedAt": "2020-11-03T20:57:00.000Z",
+            "sticky": true,
+            "id": "b7c6d7ca-6fef-406b-a911-bba873a99773",
+            "category": {
+                "organizationId": "1384d793-dae5-441c-8cc2-e78168c94018",
+                "createdAt": "2020-10-29T15:01:44.000Z",
+                "translations": [
+                    {
+                        "language": "en",
+                        "id": "4840af1e-1d74-4b4f-bdda-6e04f4569142",
+                        "text": "system-category",
+                        "type": "title",
+                        "version": 1
+                    },
+                    {
+                        "language": "fr",
+                        "id": "0bc8f49c-8f65-4319-be57-ad70cdfe144d",
+                        "text": "en français",
+                        "type": "title",
+                        "version": 1
+                    }
+                ],
+                "icon": "fa fa-envelope",
+                "id": "2569f2cc-f1d7-4664-8439-173fedfab1c3",
+                "version": 1,
+                "updatedAt": "2020-10-29T19:28:22.000Z"
+            },
+            "splash": false,
+            "updatedAt": "2020-10-29T19:27:34.462Z",
+            "status": "published"
+        }
+    ]
+}
+```
+List the notifications configured for the organization.
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`organization_id`<br/>*UUID* | The id of the organization we wish to display notifications for. When left blank, the current user's notifications will be shown.
```

Comment:
see comment in category

Reply:
65d3185551293faa443a5e59617d81963fb934f1